source/language.js

```diff
@@ -25,13 +34,8 @@ Language = Space.messaging.Serializable.extend('Language', {
 // Register as EJSON type
 Language.type('Language');
```

Comment:
@DominikGuzei: why do we still need this? Deserialization after serialization is failing without this.